modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -43,7 +42,7 @@ class GrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   {
     FILL,
     USE,
-    BYPASS
+    BYPASS,
```

Comment:
I remember asking the same question a few months back. The answer was that it makes adding new entries and reordering existing entries easier because the last entry doesn't need special treatment. Still looks odd to me, though.